python/Ganga/new_tests/TestProxy.py

```diff
@@ -69,9 +70,14 @@ def test_dict_attributes(self):
 
         # Proxy class instance
         self.assertFalse(hasattr(self.p, '_proxyClass'))
-        self.assertFalse(hasattr(self.p, '_proxyObject'))
+        self.assertTrue(hasattr(self.p, '_proxyObject'))
         self.assertTrue(hasattr(self.p, '_impl'))
 
+        # Proxied class
+        self.assertTrue(hasattr(self.p_class, '_proxyClass'))
+        self.assertFalse(hasattr(self.p_class, '_proxyObject'))
+        self.assertFalse(hasattr(self.p_class, '_impl'))
+
```

Comment:
These are 2 different things surely? One is the proxyClass of the class object the other is the class itself. (Granted if/when we get rid of the proxyclass this becomes mute as it just points to itself) I was wanting to be explicit.